ui/main.qml

```diff
@@ -32,8 +32,8 @@ ApplicationWindow {
     visible: true
 
     Component.onCompleted: {
-        setX(Screen.width / 2 - width / 2);
-        setY(Screen.height / 2 - height / 2);
+        setX(Qt.application.screens[0].width / 2 - width / 2);
+        setY(Qt.application.screens[0].height / 2 - height / 2);
```

Comment:
Is there a way to know which monitor is the "current" one? That way it opens in the same screen you clicked or something

Reply:
I don't see a way currently, for now will default to the first screen as it's common in most applications